resources/views/projects/my-projects.blade.php

```diff
@@ -18,54 +18,71 @@
         ($user->context === 'individual' &&
             ($user->individual->isConsultant() || $user->individual->isConnector()) &&
             $user->individual->isParticipant()))
-        <nav class="nav--tabbed" aria-labelledby="projects">
-            <div class="center center:wide">
-                <ul class="-mt-4 flex gap-6" role="list">
-                    @if ($user->context === 'organization')
-                        @if ($user->organization->isConsultant() || $user->organization->isConnector())
+        @if ($user->context === 'organization' ||
+            ($user->context === 'individual' && $user->individual->roles && sizeof($user->individual->roles) > 1) ||
+            ($user->indiviudal?->inProgressContractedProjects &&
+                sizeof($user->indiviudal->inProgressContractedProjects) > 0 &&
+                $user->individual?->inProgressParticipatingProjects &&
+                sizeof($user->individual->inProgressParticipatingProjects) > 0))
```

Comment:
When user->context is individual, that implies that $user->individual won't be null and using the check inside of sizeof throws an error that says sizeof can't be used on potentially null variables. To clarify, was your comment about adding checks when using  $user->individual with $user->context?

Reply:
Actually my comment was really about the last `||` section. Where you check `$user->indiviudal?->inProgressContractedProjects` is `null` (also there's a typo there). Then also check again on line 25.

Regarding `sizeof`if there is a chance it could still result in `$user->individual` being `null` you'd probably want to do something like `sizeof($user->individual?->roles ?? [])`.